lms/static/sass/features/_course-experience.scss

```diff
@@ -64,8 +64,8 @@
   }
 }
 
-// Welcome message
-.welcome-message {
+// Welcome message / Latest Update message
+.welcome-message, .update-message{
```

Comment:
NIT: would it be possible to try having the dismiss button be on the same line as the Latest Update row? cc'ed @HarryRein for suggestions here. 

![image](https://user-images.githubusercontent.com/2023680/29325804-0c0145f4-81b7-11e7-826e-b4a4a273392b.png)

hack! added:   .update-message h3 { display:inline-block; width: 90% (hack!) } 
could also throw a clearfix after the dismiss button in this case? note with my hack if you set width to less than 90% it doesn't actually clear the contents of the message... may need to wrap the actual update body in a span that clears it from the above h3 and dismiss div. ?